### DISCUSSION TITLE
Definition of Contracts/ContractOriginator is over-specific

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1366] [Foundations (FND)] [answered]
By saying "originates AND is the principal" the definition makes it unclear how to model an originator who is not a party to the final contract. Or who is a party but not the principal. 
If it is really necessary that the originator is the Principal then it should be a subclass of the latter, but I think the definition just needs to be looser and e.g. say "who is often the principal".

_Originally posted by @rivettp in https://github.com/edmcouncil/fibo/diffs/15_

**Comment by trypuz** [comment]
See [the definition of a contract originator in FIBO Viewer](https://spec.edmcouncil.org/fibo/ontology/FND/Agreements/Contracts/ContractOriginator).

**Comment by trypuz** [comment]
Any objections to @rivettp's proposal? Who will create PR?

**Comment by ElisaKendall** [comment]
So, @MikeHypercube was working with the FND FCT on the contracts ontology.  This really belongs with that work, if he still intends to integrate anything that they have done.  If he is no longer available to work on this, then I have access to a contracts attorney that could assist.

**Comment by trypuz** [comment, marked as answer]
No ContractOriginator in FIBO since 2020Q4 release.

See ContractOriginator in FIBO 2020Q3 release:
https://spec.edmcouncil.org/fibo/ontology/FND/Agreements/Contracts/ContractOriginator?version=master%2F2020Q3
